### ISSUE TITLE
User personal settings page

### ISSUE THREAD

**Issue by PVince81** [owncloud/phoenix#2056]
For many apps, we need a page where the user can specify some personal settings.

Some examples:
- mail notification settings
- whether to automatically accept incoming local shares
- oauth2 app list
- session management
- setting avatar

I think not all of them would apply to Phoenix, but for those that do, we need a place to manage this.

For now the most obvious place would be under the "Account" page, add these all as sections.
This would also require a list of quick links to be able to jump to the sections (priority depends how many settings we'll have at first)

A less obvious place would be to display those settings closer to the feature. For example sharing settings would need to appear somewhere in the files app.

**Comment by PVince81** [comment]
- [ ] extension point for Phoenix apps to define their own settings section
- [ ] add any settings that are relevant to the files app, sharing and notifications

**Comment by PVince81** [comment]
note: in the meantime as a fallback we could redirect to OC's settings page. but there is no convenient way back to Phoenix from there, so the UX will suffer.

**Comment by PVince81** [comment]
Had an idea for a shortcut to cover the time until we get there: embed the OC 10 UI into Phoenix.

See https://github.com/owncloud/phoenix/issues/2598 for a prototype

**Comment by DeepDiver1975** [comment]
Well ... following the multi url Micro framework pattern we could simply add the old Frontend by linking it in the Phoenix App menu. See #2573

**Comment by PVince81** [comment]
@DeepDiver1975 yes, but I don't want the OC 10 header to be visible for a seamless experience. See https://github.com/owncloud/phoenix/issues/2598#issuecomment-561057731
